### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by giporro**

### CODE REVIEW

File: src/main/java/it/gov/pagopa/paymentupdater/deserialize/AvroMessageDeserializer.java [hunk 1]
```diff
@@ -16,52 +17,61 @@
...
 import lombok.extern.slf4j.Slf4j;
 import tech.allegro.schema.json2avro.converter.JsonAvroConverter;
+
 @Slf4j
 public class AvroMessageDeserializer implements Deserializer<Payment> {
 
 	JsonLoader schema;
 	ObjectMapper mapper;
 	JsonAvroConverter converter;
+	Payment emptyPayment = null;
```
Comment (AleDore) [inline src/main/java/it/gov/pagopa/paymentupdater/deserialize/AvroMessageDeserializer.java:27]: What's the behavior of this?

**Review by AleDore — COMMENTED** [review]
cc: @fabriziopapi

**Comment by AleDore** [comment]
It seems that code review pipeline fails @gparente3 @giporro

File: src/main/java/it/gov/pagopa/paymentupdater/deserialize/AvroMessageDeserializer.java (hunk 1 above)
Comment (giporro) [reply src/main/java/it/gov/pagopa/paymentupdater/deserialize/AvroMessageDeserializer.java:27]: It's a typo, we remove it immediately

Comment (gparente3) [reply src/main/java/it/gov/pagopa/paymentupdater/deserialize/AvroMessageDeserializer.java]: removed

**Pull request merged**